### ISSUE TITLE
Is there an equivalent to `contract`?

### ISSUE THREAD

**Issue by sergiocorreia** [mcaceresb/stata-gtools#23]
When writing ftools, I realized that I could have `contract` for free if I just stopped at the middle of the collapse code and returned the counts of each level. Thus, the `freq` option allows you to get the same results as contract.

Is there something like that with gtools? (Running `collapse (count)` is not ideal because it requires an additional variable, it requires it to be nonmissing, etc.)

**Comment by mcaceresb** [comment]
I have added a freq option to collapse but am in the middle of implementing a dedicated command for contract, actually. The reasoning for that is that I s twofold:

1. I can contract way faster than collapse.
2. I was already implementing a toplevesof command that prints the top frequencies of contract (similar to groups; I wrote an ado file for this a while back but it was really slow).

So I have a separate internal mechanism for frequencies, percentages, etc.
